src/terminal/terminal.c

```diff
@@ -1594,7 +1598,13 @@ static int __guac_terminal_send_key(guac_terminal* term, int keysym, int pressed
         /* Non-printable keys */
         else {
 
-            if (keysym == 0xFF08) return guac_terminal_send_string(term, "\x7F"); /* Backspace */
+            /* Backspace can vary based on configuration of terminal by client. */
+            if (keysym == 0xFF08) {
+                char backspace_str[2];
+                backspace_str[0] = term->backspace;
+                backspace_str[1] = '\0';
```

Comment:
You can clean this up as:

    char backspace_str[2] = { term->backspace, '\0' };

or even:

    char backspace_str[] = { term->backspace, '\0' };

Reply:
Of course.  Done & thanks.